pkgs/development/libraries/mysocketw/default.nix

```diff
@@ -1,10 +1,20 @@
 { lib
 , stdenv
 , fetchFromGitHub
+, fetchpatch
+, fetchurl
 , openssl
 , cmake
 }:
 
+let
+
+  joinpaths-src = fetchurl {
+    url = "https://AnotherFoxGuy.com/CMakeCM/modules/JoinPaths.cmake";
```

Comment:
Might be better to fetch it from a stable URL: https://github.com/AnotherFoxGuy/CMakeCM/raw/afe41f4536ae21f6f11f83e8c0b8b8c450ef1332/modules/JoinPaths.cmake

Reply:
done in https://github.com/NixOS/nixpkgs/pull/181875